pages/guides/dwolla-js/index.mdx

```diff
@@ -1,13 +1,6 @@
 ---
 layout: guides
 title: "Overview"
-group:
-  category: Funding Sources
-  title: Dwolla.js
-guide:
-  step: 0
-  filter: get-building
-  icon: guides-icon.svg
```

Comment:
Was this the file we were talking about leaving out for now until we have a good way to implement it?

Reply:
Ah, yes good catch. I can remove that file in the cleanup ticket I'm picking up next. May discuss a few options with the team tomorrow on this topic.